cloud_experts_tutorials/cloud-experts-entra-id-idp.adoc

```diff
@@ -9,102 +9,134 @@ toc::[]
 
-When prompted, follow the prompt to enable the necessary Microsoft Graph permissions.
-
+. A dialog box appears at the top of the page. Follow the prompt to enable the necessary Microsoft Graph permissions.
++
 image:azure-portal_add-optional-claims-graph-permissions-prompt.png[Azure Portal - Add Optional Claims - Graph Permissions Prompt]
 
+[discrete]
 [id="rosa-mobb-entra-id-configure-group-claims"]
-== Configure group claims (optional)
-
-In addition to individual user authentication, OpenShift provides group claim functionality. This functionality allows an OpenID Connect identity provider, like Entra ID, to offer a user’s group membership for use within OpenShift. To enable group claims, we will configure Entra ID to provide a groups claim.
+=== Configuring group claims (optional)
 
-Next, select the "Add groups claim" button.
+Configure Entra ID to offer a groups claim.
 
+.Procedure
+. Select the *Add groups claim* button.
++
 image:azure-portal_optional-group-claims-page.png[Azure Portal - Add Groups Claim Page]
 
-Select the "Security groups" option and click the "Add" button to configure group claims for your Entra ID application.
-
+. Select the *Security groups* option and click the *Add* button to configure group claims for your Entra ID application.
++
 [NOTE]
 ====
-In this example, we are providing all security groups a user is a member of via the group claim. In a real production environment, we highly recommend scoping the groups provided by the group claim to _only those groups which are applicable to OpenShift_.
+In this example, you give all of the security groups that a user is a member of with the group claim. In a real production environment, you should scope the groups that the group claim offers to only those groups that are applicable to OpenShift.
 ====
-
++
 image:azure-portal_edit-group-claims-page.png[Azure Portal - Edit Groups Claim Page]
 
 [id="cloud-experts-entra-id-idp-configure-cluster"]
-== Configure the OpenShift cluster to use Entra ID as the identity provider
+== Configuring the OpenShift cluster to use Entra ID as the identity provider
+
+You must configure OpenShift to use Entra ID as its identity provider.
 
-Finally, we need to configure OpenShift to use Entra ID as its identity provider. While Red Hat OpenShift Service on AWS (ROSA) offers the ability to configure identity providers via the OpenShift Cluster Manager (OCM), we will use the configure the cluster's OAuth provider to use Entra ID as its identity provider via the `rosa` CLI. Before we can do so, we need to set some variables that we will use for our IDP configuration by running the following command:
+While ROSA offers the ability to configure identity providers by using {cluster-manager}, use the ROSA CLI to configure the cluster's OAuth provider to use Entra ID as its identity provider. Before configuring the identity provider, set the necessary variables for the identity provider configuration.
```

Comment:
What's the intended meaning of "while" here? There is some ambiguity in this paragraph, IMO.

ISG:

> Use with caution **while** _conjunction_
> Use while only to refer to a period of time. To avoid potential ambiguity, do not use the term as a synonym for although or though in documentation.